### PR TITLE
New styling

### PR DESCRIPTION
**Pull request by AgaK-O**
little tweaks to make it look better.

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+.heading {
+    text-align: center;
+    background: #f2f2f2;
+    padding: 20px 100px;
+    font-family: 'Oswald', sans-serif;
+    font-size: 2em;
+}
+
+#cart {
```
Comment (chmurson) [inline index.css]: hm - would that make sense to keep tipser-elements adjustments in a separate file? 

That would help us to better visualize what styles are for local bootstrap elements, and which for tipser-elements.

**Comment by chmurson** [comment]
would that be possible to get before and after screenshot ?

File: index.html [hunk 2]
```diff
@@ -1,36 +1,40 @@
...
     <!-- This is the element where we want to put Tipser cart icon (referred to with CSS selector in mountCart function) -->
-    <div id="cart" style="position: sticky; bottom: 0"></div>
+    <div id="cart"></div>
 
-    <h2 style="text-align:center">&#9759; CMS article &#9759;</h2>
+    <h2 class="heading">CMS article</h2>
     <!-- This is the element where we want to put CMS content (referred to with CSS selector in mountContent function) -->
     <div class="content-root"></div>
 
-    <h2 style="text-align:center">&#9759; Tipser product &#9759;</h2>
```
Comment (piotr-sobczyk) [inline index.html:11]: Thanks for removing hands icons! ❤️

**Comment by piotr-sobczyk** [comment]
LGTM

**Review by piotr-sobczyk — APPROVED** [review]

File: index.css (hunk 1 above)
Comment (AgaK-O) [reply index.css]: ok

**Comment by AgaK-O** [comment]
> would that be possible to get before and after screenshot ?

**Pull request merged**